src/rollup/index.ts

```diff
@@ -229,7 +229,7 @@ function checkOutputOptions(options: OutputOptions) {
 
 	if (!options.format) {
 		error({
-			message: `You must specify options.format, which can be one of 'amd', 'cjs', 'system', 'es', 'iife' or 'umd'`,
+			message: `You must specify options.format, which can be one of 'amd', 'cjs', 'system', 'esm', 'es', 'iife' or 'umd'`,
```

Comment:
Shall we already move to only documenting `esm` perhaps as a first step towards deprecation?

Reply:
Probably